user/tests.py

```diff
@@ -1,3 +1,60 @@
-from django.test import TestCase
-
-# Create your tests here.
+from django.test import TestCase
```

Comment:
please keep with convention of two initial empty lines

Reply:
yes. sorry I forget to check